Controller/Admin/Transaction/TransactionTabPostProcessing.php

```diff
@@ -31,6 +31,12 @@ class TransactionTabPostProcessing extends TransactionTab
 {
     const KEY_ACTION = 'action';
     const KEY_AMOUNT = 'amount';
+    const KEY_STATUS = 'status';
```

Comment:
Just curious: What's the point of creating these consts if they are only used by the controller but not by the view?

Reply:
Added them just so they can be reused throughout this class. There is no simple way of accessing these constants in the view template, but I don't think this is a big issue in this case.